bin/xtumlmc_build

```diff
@@ -1188,7 +1188,7 @@ sub xtuml2masl
     &xtumlmc_cleanse_for_BridgePoint ( "$i/gen/code_generation/z.xtuml", "$i/gen/code_generation/a.xtuml" );
 
     # call the utilities
-    my $sys_cmd = "cat $i/gen/code_generation/a.xtuml | x2m -i$i $command | masl $validate -i$i $command -o$out_dir";
+    my $sys_cmd = "cat $i/gen/code_generation/a.xtuml | x2m -i$i $command -k | masl $validate -i$i $command -o$out_dir";
```

Comment:
I think we need a comment here to points out the -k and that this is the place to remove it if you no longer want to output key_letter pragmas